### PR TITLE
[package_info_plus] Resolve package name linux and web

### LINKED ISSUE #203: package_info_plus for linux returns empty `packageName`
expected to return `name` from `pubspec.yaml` or, (`BINARY_NAME` or `APPLICATION_ID`) from `CMAKELists.txt`, but at least an App's identifier.

### PR DESCRIPTION
**Pull request by fabiancrx**
## Description

package_info_plus for linux and web returns now return the `package_name` contained in the `version.json` file (which was introduced in [this PR](https://github.com/flutter/flutter/pull/88457)) 
 
See the data returned on both web and linux platforms :
image

## Related Issues

Closes #203

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated the version in `pubspec.yaml` and `CHANGELOG.md`.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Comment by jpnurmi** [comment]
Looking good - just needs a rebase. I merged and published #818 first because it fixed a severe bug.

**Comment by fabiancrx** [comment]
@jpnurmi rebase finished.

File: packages/package_info_plus/package_info_plus/pubspec.yaml [hunk 1]
```diff
@@ -25,10 +25,11 @@ dependencies:
   flutter:
     sdk: flutter
   package_info_plus_platform_interface: ^1.0.2
-  package_info_plus_linux: ^1.0.4
+  package_info_plus_linux: ^1.0.5
   package_info_plus_macos: ^1.2.0
   package_info_plus_windows: ^1.0.5
-  package_info_plus_web: ^1.0.4
+  package_info_plus_web: ^1.0.5
+
```
Comment (jpnurmi) [inline packages/package_info_plus/package_info_plus/pubspec.yaml:32]: nit: extra line-break

File: packages/package_info_plus/package_info_plus_linux/CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+
+- Resolve package_name
+
 ## 1.0.4
 
 - Fix MissingPluginException
 
 ## 1.0.3
-- Add `buildSignature` to Android package info to retrieve the signing certifiate SHA1 at runtime.
+- Add `buildSignature` to Android package info to retrieve the signing certificate SHA1 at runtime.
```
Comment (jpnurmi) [inline packages/package_info_plus/package_info_plus_linux/CHANGELOG.md:10]: nit: avoid unrelated drive-by changes :)

File: packages/package_info_plus/package_info_plus_web/CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+
+- Resolve package_name
+
 ## 1.0.4
 
 - Fixed url resolving for the version.json
 
 ## 1.0.3
-- Add `buildSignature` to Android package info to retrieve the signing certifiate SHA1 at runtime.
+- Add `buildSignature` to Android package info to retrieve the signing certificate SHA1 at runtime.
```
Comment (jpnurmi) [inline packages/package_info_plus/package_info_plus_web/CHANGELOG.md:10]: ditto

File: packages/package_info_plus/package_info_plus/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+## 1.4.2
```
Comment (jpnurmi) [inline packages/package_info_plus/package_info_plus/CHANGELOG.md:1]: is it missing the respective version bump in `pubspec.yaml`?

File: packages/package_info_plus/package_info_plus_linux/CHANGELOG.md (hunk 2 above)
Comment (fabiancrx) [reply packages/package_info_plus/package_info_plus_linux/CHANGELOG.md]: My bad 😅

File: packages/package_info_plus/package_info_plus/CHANGELOG.md (hunk 4 above)
Comment (fabiancrx) [reply packages/package_info_plus/package_info_plus/CHANGELOG.md:1]: Woops, thanks for the catch !

**Review by jpnurmi — APPROVED** [review]
Thanks, LGTM :+1:
